gui/app/services/chat-messages.service.js

```diff
@@ -48,6 +48,16 @@
                 return userList;
             };
 
+            // Return User List with people in role filtered out.
+            service.getFilteredChatUserList = function() {
+                let userList = service.chatUsers;
+                userList = userList.filter(function(user) {
+                    return user.disableViewerList != null ? !user.disableViewerList : true;
+                });
+
+                return userList;
+            };
```

Comment:
We can simplify this a little bit with a few small tweaks:
1) forgo the variable assignment
2) use a fat-arrow function for the filter lamda
3) since undefined/null is considered a falsey value in js, we dont need to expressly check for it here
```ts
service.getFilteredChatUserList = function() {
    return service.chatUsers.filter((user) =>!user.disableViewerList);
};
```